test/board_test.py

```diff
@@ -0,0 +1,37 @@
+import unittest
+
+from src.board import Board
+
+class BoardTest(unittest.TestCase):
+    def test_empty_board_to_s(self):
+       board = Board()
+       expected = [ 
+            #"⊱ –––––– {⋆⌘⋆} –––––– ⊰",
+            "",
+            "         |   |   ",
+            "      ---+---+---",
+            "         |   |   ",
+            "      ---+---+---",
+            "         |   |   ",
+            "",
+            #"⊱ –––––– {⋆⌘⋆} –––––– ⊰"
+       ]
```

Comment:
It's interesting you're constructing this as an array rather than as a single string with a bunch of newlines.
A common pattern I've seen before is to make a fully form string-rendered board for the command line.
You'd be testing something like:
```
expected = """
                     |   |   
                  ---+---+---
                     |   |   
                  ---+---+---
                     |   |   
"""
```

Just a different option - but there's a million different ways.

Reply:
I chose an array because I felt it would be an easier way to keep it a bit dynamic. The client has made it clear that he may want changes in the ascii designs and whatnot and I felt that if that was the case then an array that builds the board one section at a time might be a better base to build from for the future.